templates/index.html

```diff
@@ -1,98 +1,272 @@
+
 <!DOCTYPE html>
-<html>
-<head>
-	<meta charset="utf-8">
-	<title>How Are You Really?</title>
-	<script src="static/lib/jquery-3.6.0.min.js"></script>
-	<style>
-		#video {
-			width: 500px;
-			height: 375px;
-			background-color: #666;
-		}
-		#canvas {
-			display: hidden
-		}
-	</style>
+<html lang="en">
+	<head>
+		<script src="static/lib/jquery-3.6.0.min.js"></script>
+		<meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"/>
+    <link rel="stylesheet" href="static/lib/style.css">
```

Comment:
Please only keep one of the `style.css` and remove link to the other file.

Reply:
One of the CSS file is for testing (the one in templates) and the other CSS file (the one in static) is for when running flask. For some reason for me to open and test in default browser, I have to have the CSS within the local folder. And for flask it only registers the file in the static folder.